src/ast/rewriter/datatype_rewriter.cpp

```diff
@@ -21,33 +21,8 @@ Module Name:
 br_status datatype_rewriter::mk_app_core(func_decl * f, unsigned num_args, expr * const * args, expr_ref & result) {
     SASSERT(f->get_family_id() == get_fid());
     switch(f->get_decl_kind()) {
-    case OP_DT_CONSTRUCTOR: {
```

Comment:
datatype_rewriter is used from other code paths than QEL.
Why would this code be deleted?

Reply:
the latest commit ([0e78d24](https://github.com/Z3Prover/z3/pull/6820/commits/0e78d240b66f17352f2bbe283228d49d221df008)) moves the rewrite into `qe_mbp`